ibis/spark/tests/test_udf.py

```diff
@@ -0,0 +1,374 @@
+import pandas as pd
+import pandas.util.testing as tm
+import py4j
```

Comment:
Remember that this is an optional dependency. You need to use `importorskip` here.

Reply:
Also, remember to check the CI periodically to see if there are failures.